gemfire-core/src/main/java/com/gemstone/gemfire/internal/cache/EntryEventImpl.java

```diff
@@ -1823,13 +1823,13 @@ private void acquireMemory(final LocalRegion owner,
       if (this.memoryTracker != null) {
         owner.acquirePoolMemory(oldSize,
                 event.getNewValueBucketSize(),
-                true,
+                false,
                 this.memoryTracker,
                 true);
       } else {
         owner.delayedAcquirePoolMemory(oldSize,
                 event.getNewValueBucketSize(),
-                true,
+                false,
                 true);
       }
```

Comment:
@rishitesh why entry overhead excluded for accounting in puts?

Reply:
@sumwale that is for update. Already that overhead is accounted in a create entry.

Reply:
@rishitesh shouldn't this then be "!isUpdate" rather than always false or true? AFAIK EntryEventImpl.setNewValueInRegion is invoked for new inserts as well as updates.

Reply:
@sumwale That's correct. The code block is executed only in case of isUpdate = true. for isUpdate= false entry overhead is accounted.

Reply:
Oh, ok. But then the block below should have it instead of true for the "wasTombstone==true" case. Doesn't look like accounting is being skipped for tombstone case here. If yes, then the "wasTombstone" flag should be removed from this method.